content/22.vaccines.md

```diff
@@ -470,6 +469,76 @@ Similar to BNT162b2, the vaccine was associated with mild-to-moderate adverse ef
 In late 2020, both vaccines both received approval from the United States's Food and Drug Administration (FDA) under an emergency use authorization [@url:https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19; @url:https://www.cdc.gov/mmwr/volumes/69/wr/mm695152e1.htm], and these vaccines have been widely distributed, primarily in North America and the European Union [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html].
 As the first mRNA vaccines to make it to market, these two highly efficacious vaccines demonstrate the power of this emerging technology, which has previously attracted scientific interest because of its potential to be used to treat non-infectious as well as infectious diseases.
 
+### Viral Evolution and Vaccine Efficacy
+
+An ongoing topic of concern is how the evolution of SARS-CoV-2 influences vaccine efficacy.
+Between December 2020 and April 2021, one prospective cohort study obtained weekly nasal swabs from 3,975 individuals at high risk of SARS-CoV-2 exposure (health care workers, frontline workers, etc.) within the United States [@doi:10.1056/NEJMoa2107058].
+Among these participants, 3,179 (80%) had received at least one dose of an mRNA vaccine and 2,686 (84%) were fully vaccinated.
+For each vaccinated participant (defined here as having received at least dose 1 more than 7 days ago) whose sample tested positive for SARS-CoV-2, they categorized the viral lineage(s) present in the sample as well as in samples from 3-4 unvaccinated individuals matched by site and testing date.
+Overall efficacy of mRNA vaccines was estimated at 91% with full vaccination, similar to the reports from the clinical trials.
+The occurrence of fevers was also lower in individuals who were partially or fully vaccinated, and the duration of symptoms was approximately 6 days shorter.
+<!--Flesh this out, what were the relative rates of strains?-->
+
+### Vaccines and Community Spread
+
+The vaccine clinical trial data demonstrate a significant reduction in the likelihood of contracting symptomatic COVID-19.
+However, given that vaccination rates are unlikely to reach 100%, another concern is whether vaccinated individuals with or without symptoms of COVID-19 can still spread SARS-CoV-2.
+This question is made up of several components.
+The crux is whether vaccinated individuals with a SARS-CoV-2 infection, regardless of symptom status, are as contagious as unvaccinated, infected individuals.
+Additionally, as outlined above, an important qualification is that the variants of SARS-CoV-2 circulating at the time of each study must be considered in light of the effect of evolution on vaccine efficacy.
```

Comment:
In this area, I think its important to keep in mind that the main goal of a vaccine is to prevent severe illness and death. The added bonus is sometimes you get reduced infectivity and reduced disease transmission... but that isn't the ultimate goal of a vaccine. Hence, why we have "breakthrough" infections, but notably limited deaths form those infections. We seem to have raised the standards of what a vaccine should do, when in reality we haven't done that for any other vaccine. I don't know how we include that here somewhere, maybe it is something to keep in mind to fit in a discussion section.

Reply:
This is a very good point. I agree. I remember studying the evolution of flu, one of the big questions in the field was to predict its mutation that would result in needing to update the vaccine. I know that when there a "jump" meaning the virus was circulating in pigs or birds and then jumped into humans, those outbreaks have been more difficult compared to gradual mutations, which is maybe what @RLordan is referring to about the "breakthroughs".

Reply:
Should we define what a "breakthrough infection" means?